### DISCUSSION TITLE
Could not resolve to an issue a moved issue

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#81341] [Mobile] [answered]
GitHub Mobile

1. Get a GitHub Mobile notification about a comment to an issue I filed 
2. Tap it

![Screenshot_20231221-075202.png](https://github.com/community/community/assets/2119212/1d5f6df4-f93b-42ee-80f9-8bfb7bdd32cf)

Expected results: since it technically exists and hasn't been deleted, opening the notification should work. 

Steps to prove it still works:
1. Open https://github.com/github/docs/issues/30780#issuecomment-1865095578 in a web browser
2. Get redirected to https://github.com/github/rest-api-description/issues/3259#issuecomment-1865095578

**Comment by kangarruu** [comment, marked as answer]
@jsoref thank you for letting us know! I'll open up an internal issue to track this. Just to verify, you started experiencing this error after the linked issue was transferred to another repository?

**Reply by jsoref** [reply]
Correct. And the url works fine in the browser.

The problem is likely that the mobile app is being overly cautious by asking a very specific question to the github api to which the github api likely says "nah, that isn't really a live ticket here" whereas the normal request that a browser makes yields a 302 which the browser merrily follows.

**Reply by jsoref** [reply]
Also note that you can trigger it by just triggering the url from other places while the mobile app is set as the url handler for the url, you don't need to get a notification to reproduce.

**Reply by kangarruu** [reply]
Great thanks for verifying 👍🏻, I've updated the tracking issue and escalated it internally. We apologize for any inconvenience this may be causing!
